packages/types/src/Metadata/v6/Constants.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright 2017-2019 @polkadot/types authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import Bytes from '../../primitive/Bytes';
+import Text from '../../primitive/Text';
+import Type from '../../primitive/Type';
+import Struct from '../../codec/Struct';
+import Vector from '../../codec/Vector';
+
+export class ModuleConstantMetadata extends Struct {
+  constructor (value?: any) {
+    super({
+      name: Text,
+      ty: Text,
+      value: Bytes,
```

Comment:
yeah, we can leave Bytes now. To be perfectly correct, it should be `value: ty` (i.e. value should be the type of what was returned inside ty). But we do the same for `fallback: Bytes` right now, so let's keep it like this.